www/default.js

```diff
@@ -143,9 +143,11 @@ function change_cost(duration) {
     per_time = elem.data("pricing")[g_settings.region];
     if (per_time && !isNaN(per_time)) {
       per_time = (per_time * multiplier).toFixed(3);
-      elem.text("$" + per_time + " " + duration);
+      elem.empty();
+      elem.append('<span sort="' + per_time + '">$' + per_time + ' ' + duration + '</span>');
```

Comment:
I don't understand why did you replace the single elem.text with the pair of elem.empty/elem.append.